src/taskpane/components/ReferenceList.tsx

```diff
@@ -11,15 +11,22 @@ import {
 } from "@fluentui/react";
 import { MetaData } from "citeproc";
 import React, { ReactElement } from "react";
+import EditCitation, { citationMetaData } from "./EditCitation";
 
 const theme: ITheme = getTheme();
 const { palette, semanticColors, fonts } = theme;
 export interface bib extends MetaData {
   isSelected: boolean;
+  label: string;
```

Comment:
Note that these information are specific to a particular citation of a document, and not an addition to the metadata of a document. So you can cite the same document in two different locations with different suffix/prefix/.... So maybe it would be better to extract this in a new interface.